src/DevBetterWeb.Web/Pages/Admin/UserReports/Cancellations.cshtml.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using DevBetterWeb.Core;
+using DevBetterWeb.Core.Entities;
+using DevBetterWeb.Core.Interfaces;
+using DevBetterWeb.Core.Specs;
+using DevBetterWeb.Core.ValueObjects;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace DevBetterWeb.Web.Pages.Admin.UserReports
+{
+  [Authorize(Roles = AuthConstants.Roles.ADMINISTRATORS)]
+
+  public class CancellationsModel : PageModel
+  {
+
+    private static readonly DateTimeRange defaultRange = new DateTimeRange(DateTime.Today.AddDays(-29), DateTime.Today.AddDays(-1));
```

Comment:
Understand this will show results through end of day-before-yesterday, not through yesterday, because of midnight default on datetimes. Is this what you want here?